spec/lib/pre_assembly/bundle_spec.rb

```diff
@@ -108,17 +108,7 @@
     it 'handles containers correctly' do
       expect(multimedia.digital_objects.first.container.size).to be > multimedia.bundle_dir.size
     end
-  end
-
-  describe '#load_checksums' do
-    it 'loads checksums and attach them to the ObjectFiles' do
-      multimedia.send(:all_object_files).each { |f| expect(f.checksum).to be_nil }
-      multimedia.digital_objects.each { |dobj| multimedia.load_checksums(dobj) }
-      multimedia.send(:all_object_files).each { |f| expect(f.checksum).to match(md5_regex) }
-    end
-  end
 
-  describe '#digital_objects' do
```

Comment:
i just collapsed the two sets of `#digital_objects` tests into one describe block.